extensions/amazon-lambda/runtime/src/main/java/io/quarkus/amazon/lambda/runtime/LambdaConfig.java

```diff
@@ -19,6 +19,14 @@
      *
      */
     @ConfigItem
-    Optional<String> handler;
+    public Optional<String> handler;
+
+    /**
+     * If true, this will enable the aws event poll loop within a Quarkus test run. This loop normally only runs in native
+     * image. This option is strictly for testing purposes.
+     *
+     */
+    @ConfigItem
+    public boolean enablePollingJvmMode;
```

Comment:
Do we need this to be config, can't we handle this transparently? e.g. have the io.quarkus.amazon.lambda.test.LambdaResourceManager set a system property, and just look at the system property to decide if we are in a test environment.

Reply:
Either way it is not worth holding up the PR over it.

Reply:
I'll try that out in another PR.